index.html

```diff
@@ -281,5 +281,58 @@ <h2>Skills</h2>
         </ul>
       </div>
     </section>
+    <footer class="contact">
+      <img src="./Images/Ellipse-2.png" alt="Elipse" class="Elipse-2" />
+      <img src="./Images/Rectangle-1.png" alt="Rectangle" class="Rectangle-1" />
+      <img src="./Images/Ellipse-3.png" alt="Elipse" class="Elipse-3" />
+      <img src="./Images/Rectangle-2.png" alt="Rectangle" class="Rectangle-2" />
+      <img src="./Images/Ellipse-1.png" alt="Elipse" class="Elipse-1" />
+      <h1 class="contact-title">Contact me</h1>
+      <p class="contact-description">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+      <form
+        action="https://formspree.io/f/xvollyol"
+        class="contact-form"
+        method="POST"
+      >
+        <ul class="inputs">
+          <li>
+            <input
+              type="text"
+              value="Faranosh"
+              required
+              placeholder="Enter Your Name"
+              maxlength="30"
+              id="name"
+            />
```

Comment:
Both of your input fields and your button have a border that is not present in the template design.

Here's your design:

<img width="247" alt="Screen Shot 2022-06-01 at 8 09 52" src="https://user-images.githubusercontent.com/82426682/171412148-4f63f590-a92f-4ed8-a23c-f2e2f2ae46dd.png">

Here's the template design:

<img width="251" alt="Screen Shot 2022-06-01 at 8 10 25" src="https://user-images.githubusercontent.com/82426682/171412249-b7bd5b09-8425-42cb-b328-40181d0aa741.png">
(I apologize about the green border, some fellow students were using Figma when I took the screenshot).

Applying a `border: none` should be enough to fix this. Kindly do so as following the template design as close as possible is a project requirement.

<img width="895" alt="Screen Shot 2022-06-01 at 8 12 03" src="https://user-images.githubusercontent.com/82426682/171412531-cedfb041-5f79-4515-9785-ec4122325426.png">